app.js

```diff
@@ -127,6 +145,14 @@ function handleResponse(res, err, apiResponse){
   }
 };
 
+function toString(token){
+  return "[A:" + token.access_token + " R:" + token.refresh_token + " X:" + token.expires_at + "]";
+}
+
+function log(message){
```

Comment:
Also the vogue is to have structured logging, so rather than a message, return an object of attributes (one of which might be a message)